A/ASL/build_tarballs.jl

```diff
@@ -0,0 +1,92 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "ASLBuilder"
+version = v"0.1.0"
+
+# Collection of sources required to build ThinASLBuilder
+sources = [
+    "http://netlib.org/ampl/solvers.tgz" =>
+    "775b92cadaf95af73fdeec3effba6b9c6ffdc518e5f628b575140fb170885903",
+    "./bundled"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd solvers/
+mkdir $prefix/lib
```

Comment:
The environment variable `${libdir}` points to the right directory on all platforms, that is `${prefix}/bin` for Windows and `${prefix}/lib` everywhere else:
```suggestion
mkdir ${libdir}
```